README.md

```diff
@@ -59,6 +59,8 @@ You can specify the versions by queries (case insensitive):
 Browserslist works with separated versions of browsers.
 You should avoid queries like `Firefox > 0`.
 
+All queries are based on the [Can I use](caniuse.com) support table, e. g. `last 2 iOS versions` might select `8.4, 9.2, 9.3` (mixed major & minor), whereas `last 2 Chrome versions` might select `50, 49, 48` (major only).
```

Comment:
Umm I find this a bit confusing... Shouldn't `last 2 versions` only match 2 versions instead of 3?